lib/uname.js

```diff
@@ -0,0 +1,43 @@
+var getUname = require('node-uname')
+var decodeArguments = require('./common').decodeArguments
+
+module.exports = uname
+
+function uname(args) {
+    var params = decodeArguments(args)
+    var info = getUname()
+
+    if (params.a || params.all) {
+        return [info.sysname, info.nodename, info.release, info.version, info.machine].join(' ')
```

Comment:
All the library functions return the "raw" data, in this case `info`. The args that only apply to the "human oriented" output should be put inside an [.inspect()](https://nodejs.org/api/util.html#util_custom_inspect_function_on_objects) method. Could you be able to change it?